lib/src/modules/renameInput/RenameInputForm.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+
+export function RenameInputForm({
+  stopEditing,
+  onRename,
+  label,
+  text,
+  id,
+  multiline,
+  classes,
+  placeholder
+}) {
+  const [value, setValue] = useState(text);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    if (value.trim() !== '') {
+      onRename(value);
+    }
+    stopEditing();
+  };
+
+  const handleOnChange = e => {
+    setValue(e.target.value);
+  };
+
+  const handleEsc = e => {
+    const isEscKey = e.keyCode === 27;
+    const isEnterKey = e.keyCode === 13;
+    if (isEscKey) {
+      stopEditing();
+    }
+    if (isEnterKey) {
+      handleSubmit(e);
```

Comment:
Even though we're in a form we still need to do this incase we're using a textarea

Reply:
Do we need the `onSubmit` attribute on the form if it is handled here then?

Reply:
As this is in a form we shouldn't need to do this.

Reply:
@kyleharper I didn't think we would either – it works great if its just a text input, but like I said it doesn't work if it is a textarea. If its a textarea it just adds new lines rather than submitting, something in textarea must override forms enter key handling!